internal/providers/terraform/aws/route53_resolver_endpoint.go

```diff
@@ -1,86 +1,20 @@
 package aws
 
 import (
+	"github.com/infracost/infracost/internal/resources/aws"
 	"github.com/infracost/infracost/internal/schema"
-	"github.com/infracost/infracost/internal/usage"
-
-	"github.com/shopspring/decimal"
 )
 
-func GetRoute53ResolverEndpointRegistryItem() *schema.RegistryItem {
+func getRoute53ResolverEndpointRegistryItem() *schema.RegistryItem {
 	return &schema.RegistryItem{
 		Name:  "aws_route53_resolver_endpoint",
 		RFunc: NewRoute53ResolverEndpoint,
 	}
 }
-
 func NewRoute53ResolverEndpoint(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
-	region := d.Get("region").String()
-
-	resolverEndpointCount := int64(len(d.Get("ip_address").Array()))
-
-	costComponents := []*schema.CostComponent{
-		{
-			Name:           "Resolver endpoints",
-			Unit:           "hours",
-			UnitMultiplier: decimal.NewFromInt(1),
-			HourlyQuantity: decimalPtr(decimal.NewFromInt(resolverEndpointCount)),
-			ProductFilter: &schema.ProductFilter{
-				VendorName:    strPtr("aws"),
-				Region:        strPtr(region),
-				Service:       strPtr("AmazonRoute53"),
-				ProductFamily: strPtr("DNS Query"),
-				AttributeFilters: []*schema.AttributeFilter{
-					{Key: "usagetype", ValueRegex: strPtr("/ResolverNetworkInterface/")},
-				},
-			},
-		},
-	}
-
-	queryTierLimits := []int{1000000000}
-
-	if u != nil && u.Get("monthly_queries").Exists() {
-		monthlyQueries := decimal.NewFromInt(u.Get("monthly_queries").Int())
-		dnsQueriesTier := usage.CalculateTierBuckets(monthlyQueries, queryTierLimits)
-		tierOne := dnsQueriesTier[0]
-		tierTwo := dnsQueriesTier[1]
-
-		if tierOne.GreaterThan(decimal.NewFromInt(0)) {
-			costComponents = append(costComponents, dnsQueriesCostComponent(region, "DNS queries (first 1B)", "0", &tierOne))
-		}
+	resolverEndpoints := int64(len(d.Get("ip_address").Array()))
```

Comment:
How about we rename it to `resolverEndpointsLen`?

Reply:
I've updated to `resolverEndpointsCount` but still kept the struct property as `ResolverEndpoints` so it's consistent with what we do for (most) other integer properties and in the usage file.